pkg/gitweb

```diff
@@ -0,0 +1,44 @@
+[mirrors]
+http://cdn.kernel.org/pub/software/scm/git/git-2.22.0.tar.xz
+https://mirrors.edge.kernel.org/pub/software/scm/git/git-2.22.0.tar.xz
+
+[vars]
+filesize=5609772
+sha512=75b3dcac36f80281effcd099944de34050a35f3599ce42f86ce60455b6c952039fb0f6438d296e0cc9c0651d4a17f467780dc475669227d3c98ddefe91723d42
+pkgver=18
+desc='git web interface'
+
+[deps]
+musl
+curl
+
+[deps.run]
+perl
+
+[build]
+cd gitweb || exit 1
+
+SRVDIR=/var/www
+REPOS=/home/gitweb
+
+if [ ! -d "$REPOS" ]; then
+	mkdir -p "$REPOS" || exit 1
+fi
+
+make -j$MAKE_THREADS \
+	GITWEB_PROJECTROOT="/home/gitweb" \
+	prefix=/
```

Comment:
in packages, we use $butch_prefix since this a setting a user can modify.
some build systems have trouble with an empty prefix, in such cases we add something like `[ -z $butch_prefix ] && butch_prefix=/` to the top of the build section